airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_stream.py

```diff
@@ -172,3 +173,32 @@ def get_cursor(self) -> Optional[Cursor]:
         if self.retriever and isinstance(self.retriever, SimpleRetriever):
             return self.retriever.cursor
         return None
+
+    def _get_checkpoint_reader(
+        self,
+        logger: logging.Logger,
+        cursor_field: Optional[List[str]],
+        sync_mode: SyncMode,
+        stream_state: MutableMapping[str, Any],
+    ) -> CheckpointReader:
+        """
+        This method is overridden to skip classification for incremental streams
```

Comment:
Can we explain why we want to skip the classification? If feels like this is to avoid part of the parent behavior which is problematic. What I don't understand is why we keep this behavior for the parent if this is problematic